MMR.UI/Forms/CustomItemListEditForm.cs

```diff
@@ -20,6 +20,7 @@ public partial class CustomItemListEditForm : Form
         public string ExternalLabel { get; private set; }
         public List<Item> ItemList { get; private set; } = new List<Item>();
         public string ItemListString { get; private set; }
+        public Func<Item, string> _labelSelector { get; private set; }
```

Comment:
This either shouldn't be `public` or should be written as `LabelSelector`. Since it's not used outside the class, it should probably just be private.